pkg/chunk/gcp/bigtable_object_client.go

```diff
@@ -65,6 +68,10 @@ func (s *bigtableObjectClient) PutChunks(ctx context.Context, chunks []chunk.Chu
 
 	for tableName := range keys {
 		table := s.client.Open(tableName)
+		if s.writeLimiter.Allow() == false {
```

Comment:
Using `Allow()` suffers the issue I've already commented about, reason why you've changed the `pkg/chunk/gcp/bigtable_index_client.go` implementation.

Reply:
okay! Made similar changes as bigtable_index_client go. Since, we no longer have the ability to write metrics for these rate limiters, should I remove the metrics altogether?